app/components/widgets/ProgressSteps.js

```diff
@@ -48,7 +57,7 @@ export default class ProgressSteps extends Component<Props> {
       } else if (idx === currentStep) {
         // for current step, 0 = LOAD | 1 = PROCESS | 9 = ERROR
         // 0 = LOAD and 1 = PROCESS has same icon but for 9 = ERROR there is a error icon
-        displayIcon = (progressInfo.stepState === 9) ? 'error' : 'none';
+        displayIcon = (stepState === 9) ? 'error' : 'none';
```

Comment:
This is not from you, but I think is kind of ugly to have comparison by numbers. Also having to re-explain something that is declared is strange.

Is it possible to do something like this?

```suggestion
        displayIcon = (stepState === StepState.ERROR.valueOf()) ? 'error' : 'none';
```